sql/dolphinscheduler_postgre.sql

```diff
@@ -303,7 +303,7 @@ CREATE TABLE t_ds_process_definition (
   tenant_id int DEFAULT '-1' ,
   create_time timestamp DEFAULT NULL ,
   update_time timestamp DEFAULT NULL ,
-  PRIMARY KEY (id) ,
+  PRIMARY KEY (id, code) ,
```

Comment:
Is it more appropriate to add a unique index to "code".

Reply:
Thanks for your review.
I made this change by folllowing the same table schema in mysql:https://github.com/apache/dolphinscheduler/blob/5d3b75a0a32da02ad562386a8253727ab0dc6b75/sql/dolphinscheduler_mysql.sql#L411 .  I've also update the sql upgrade script.
I'm not sure what's the best way to make the scheme's upgrade 😭

Reply:
Primary key remove "code".
Add a unique index to "code".

Reply:
@wen-hemin  thanks! done this:
mysql:

![image](https://user-images.githubusercontent.com/52202080/121639637-6c54e780-cabf-11eb-8af8-2abf611e40d7.png)
postgresql:

![image](https://user-images.githubusercontent.com/52202080/121639704-842c6b80-cabf-11eb-94c0-69f93b85c1bc.png)

Reply:
Thanks @wen-hemin 
you mean **the primary key should be `id` instead of `(`id`,`code`)`?**
If so, the schema defined early by other developers in  `dolphinscheduler_mysql.sql`(https://github.com/apache/dolphinscheduler/blob/5d3b75a0a32da02ad562386a8253727ab0dc6b75/sql/dolphinscheduler_mysql.sql#L411) is wrong. Can you confirm this? I'm not familiar with the relevant background

Reply:
Yes, no problem. Code adds a unique index.

Reply:
Thanks @wen-hemin , I have resubmitted it, please review again.